src/components/SkeletonLoader/SkeletonLoader.stories.mdx

```diff
@@ -101,7 +113,26 @@ to better view the example. The default value is `100%`.
 
 <ArgsTable story="SkeletonLoader with Controls" />
 
-# Card Placeholders in a Grid
+## Accessibility
+
+According to W3C's success criterion 2.2.2 for operable user interfaces:
+
+> For any moving, blinking or scrolling information that (1) starts automatically,
+> (2) lasts more than five seconds, and (3) is presented in parallel with other
+> content, there is a mechanism for the user to pause, stop, or hide it unless
+> the movement, blinking, or scrolling is part of an activity where it is essential;
+
+The `SkeletonLoader` component can be used as a placeholder for local content
+that is loaded in parallel but it typically does not last more than five (5)
```

Comment:
If it lasts more than five seconds would it be in violation of this rule?

Reply:
Yes.